webxr-api/view.rs

```diff
@@ -90,3 +90,40 @@ pub enum Views {
     Mono(View<Viewer>),
     Stereo(View<LeftEye>, View<RightEye>),
 }
+
+#[derive(Clone, Copy, Debug)]
+#[cfg_attr(feature = "ipc", derive(Serialize, Deserialize))]
+pub struct ClipPlanes {
+    pub near: f32,
+    pub far: f32,
+    /// Was there an update that needs propagation to the client?
+    update: bool,
```

Comment:
Hmm, it would be nice if we didn't have to keep a dirty bit on the `ClipPlanes` itself.